pkg/reconciler/kubernetes/tektonhub/tektonhub.go

```diff
@@ -164,6 +174,44 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, th *v1alpha1.TektonHub)
 	return nil
 }
 
+func (r *Reconciler) manageUiComponent(ctx context.Context, th *v1alpha1.TektonHub, hubDir, version string) error {
+	if err := r.validateUiConfigMap(ctx, th); err != nil {
+		th.Status.MarkUiDependencyMissing("UI config map not present")
+		r.enqueueAfter(th, 10*time.Second)
+		return err
```

Comment:
we can skip returning err here if we have enqueueAfter

Reply:
or we could add error as part of status
```
th.Status.MarkUiDependencyMissing("UI config map not present: %v", err)
```

Reply:
yeah when [this](https://github.com/tektoncd/operator/pull/645) patch gets merged, i'll soon update after this